runtime/permissions.rs

```diff
@@ -302,6 +302,9 @@ pub struct FfiDescriptor(pub PathBuf);
 
 impl UnaryPermission<ReadDescriptor> {
   pub fn query(&self, path: Option<&Path>) -> PermissionState {
+    if self.global_state == PermissionState::Granted {
```

Comment:
Does revoking permissions get rid of the global permission grant?

Reply:
yes https://github.com/denoland/deno/blob/5f21b3b951143d66946f4e043c0d21e96bb84c18/runtime/permissions.rs#L383